### PR TITLE
:scissors: remove leaflet images

### LINKED ISSUE #213: Use CDN to host Leaflet images
We're using a CDN to serve Esri-Leaflet and I'm wondering if it's still necessary for us to host the icon images for Leaflet locally (https://github.com/koopjs/koop/tree/master/public/images).

A little investigation shows that the Leaflet CDN does not host them, but [cdnjs](https://cdnjs.com/) does (https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/images/layers.png). cdnjs also supports HTTPS, another thing missing from Leaflet CDN. I suggest we just switch over to cdnjs and remove the images folder as 0.7.3 is very stable and cdnjs isn't going anywhere. We'll need to make sure to point to the right image in any scripts like https://github.com/koopjs/koop/blob/master/public/js/map.js#L24.

### PR DESCRIPTION
**Pull request by jgravois**
closes #213 

refactored code for breaking changes in esri-leaflet 1.0.0
got rid of leaflet images and switched to using `L.Icon.Default()`

### CODE REVIEW

**Comment by dmfenton** [comment]
probably need to rebase this on master

**Comment by jgravois** [comment]
fixed merge conflict.  happy to squash when everyone's happy with everything else.

**Comment by dmfenton** [comment]
:+1:

File: public/js/map.js [hunk 1]
```diff
@@ -11,19 +11,17 @@ function koopMap (dom) {
...
+      url: url,
       pointToLayer: function (geojson, latlng) {
         return L.marker(latlng, {
-          icon: L.icon({
-            iconUrl: '/images/marker-icon.png',
-            iconRetinaUrl: '/images/marker-icon-2x.png',
-            iconSize: [20, 30],
+          icon: new L.Icon.Default({
             iconAnchor: [10, 25],
             popupAnchor: [0, -20]
```
Comment (ngoldman) [inline public/js/map.js]: @jgravois  does `L.Icon.Default` not include these anchor properties already?

Comment (jgravois) [inline public/js/map.js]: yeah, we're tweaking the values slightly though
https://dl.dropboxusercontent.com/u/59331579/js/default-icon.html

**Pull request merged**